Attempt at redesign

almet opened the pull request:
I've made some changes on the website design. I believe this looks better, but obviously this is a matter of personal taste.

![image](https://github.com/user-attachments/assets/66bb3512-2f9c-487e-a4f8-05f108c73cb6)

Some changes I did:

- There is no linear gradient on the main page, it's now a white background
- I removed the "small caps" titles

I'm very curious about your thoughts about these changes, so I'm all ears :-) If you don't want to build it locally I put a copy of it at http://dzrocks.notmyidea.org/

eloquence commented:
I tried a couple of variants generated via Gemini 2.5 Pro, and was able to arrive at one that I quite like, which I pushed to the `redesign2` branch if you want to compare. I'd still need to audit the model's output a bit more before opening a PR, but I do like the color palette, the margins on larger screen resolution, and the tightening of whitespace. (The download links look a bit cleaner in the current design to me, those would need some work I think). What do you think of this direction?

![Screenshot 2025-03-27 at 23-25-22 Dangerzone Convert potentially dangerous documents into safe PDFs](https://github.com/user-attachments/assets/980bf9d2-ddb5-4d88-8fb5-0c3483417367)

apyrgio commented:
First of all, kudos Alexis for kickstarting this :heart: .

I just wanted to chime in and say that I like Alexis' idea with the white background and the orange accents. I find they tie in stylistically with FPF's "redacted" aesthetic. I agree with Erik though that there are some nuances with this design, so what I'd like to see is basically Erik's updated version, but with the original white background, and some orange touches here and there (e.g., "potentially <orange>dangerous</orange> PDFs"). But when selecting the text, the highlight color should be the default one and override our stylistic touches.

almet commented:
Thanks for your feedback. I'll try to update this PR, incorporating your comments; they all make sense :-)

Especially, with some more (well, one day) perspective, I feel that the orange background for the titles might not be that nice. Also, I want to try adding a background to the main area (the one with too much white background), but I'm not sure which image would fit here. Just something to add some texture maybe. I'll see.

About your proposal @eloquence, thanks. I like how the download links are better there, and yeah, the color contrast need some more work on the proposal I made.

Also, the design feels a bit constrained, with a lot of structured sections (boxes in boxes) that make it seem compartmentalized, and I find the color palette somewhat formal.

I'll update this issue when I have incorporated your comments!

harrislapiroff commented:
Thanks for making this redesign, Alexis! I like the minimal white, but agree the orange background on the titles is a bit much. Maybe a light orange background with dark text would feel more seamless?

Also agree that as long as we're in here, we might as well tighten up the whitespace. I like that the first section is a bit spacious but there's no need for it to take a full screen of space.

And let's not let perfect be the enemy of good here. I think this is an improvement on what we have now already and with a few tweaks, it's mergeable!

almet commented:
Here is an updated version, that you still find at https://dzrocks.notmyidea.org.

Here's a capture:
![image](https://github.com/user-attachments/assets/60f4a885-25bf-4f53-b377-b6b882a1d3c8)

I did the following:

- Reduced the spacing after the introduction block
- Added unicode logos for the features section
- Changed the colors of the button
- Removed the orange color for the headers
- Replaced the Linux logo with another one
- Put the Tails logo on white background with rounded corners.

Let me know what you think :-)

eloquence commented:
I really like this revised version @almet. A couple of small nits:
- The navigation links currently change color as you click on them - they're black-on-white on the main page, and orange-on-white on the other pages. This might be a holdover from the old design.
- The contrast between the orange links and the white background may be a bit too low from an accessibility standpoint. At least per https://webaim.org/resources/contrastchecker/ if I enter `#F97B25` on `#FFFFFF`, it reports 2.66:1 which does not quite pass WCAG at an AA level.

Otherwise, this LGTM as an iterative improvement. I think aligning things to a simple grid would help as a next step -- making sure that the different elements (navigation, footer, "features" block etc.) align with consistent margins on the left and right. But I think that could be done as a follow-up PR.

almet commented:
Thanks, I've updated the proposal with the following changes:

- I used the black-on-white links for the menu everywhere, it was indeed a leftover from the previous design, where the main page had the linear gradient background ;
- I updated the color of the links, and I've actually had a hard time to find something that matches our current color palette and doesn't break accessibility, so I stayed with... black links. It lacks a bit of fun, I think but I don't have any other idea for now.

eloquence commented:
@almet In case that's not intentional, just a note that these changes haven't been pushed yet. But yeah, that makes sense to me :)

almet commented:
Sorry, they were one `git push` away, just pushed them, it wasn't intentional, thanks for pointing it out. Let me know if that's okay for you!

apyrgio commented:
I like it! Feel free to merge.
